src/Components/EditProduct/EditForm.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import useForm from './useForm'
+import {Col, Row, Button } from 'react-bootstrap'
+
+const EditForm = () => {
+	const { handleChange, values } = useForm();
+	return(
+		
+		<Col >
+			<div className="form-container">
+				<form className="product-form">
+				<Row>

```

Comment:
It is <Row><Col></Col></Row> in Bootstrap. You are having it the other way around so you may have troubles to align this nicely. 

https://react-bootstrap.github.io/layout/grid/

Reply:
https://getbootstrap.com/docs/4.0/components/forms/